src/flush.rs

```diff
@@ -70,6 +70,7 @@ impl DbInner {
             self.flush_imm_wal_to_memtable(wguard.memtable(), imm.table());
             self.maybe_freeze_memtable(&mut wguard, imm.id());
             imm.table().notify_flush();
+            imm.table().notify_durable();
```

Comment:
Since notify flush is meant to be used only from the context of backpressure - I've moved it to the ImmutableMemtable type and changed the name to `await/notify_flush_to_l0`.